### DISCUSSION TITLE
How do you use correspondent and document type?

### DISCUSSION THREAD

**Discussion by Mannshoch** [jonaswinkler/paperless-ng#1398] [General]
Need a bit inspiration for a proper use. May you could help me with some experience. 

I manytimes have unique Corrspondents, how do you handle that? Create them manually have a "unique" correspondent or do you not set them?

similar Quest for Documtent type. What document type dou you normally use?

Ar there moments where you modify the PDF content of a new content? I sometime remove all empty breaks and correct some texterrors.

**Comment by TobenderZephyr** [comment]
I'm fairly new to paperless documentation, but that's what I was doing so far:

Add correspondents for every organization, I will definetely hear from the following years:
- bank accounts
- insurance companies
- internet / tv / mobile phone / estate contracts
- Amazon and other subscriptions

All of those have either regular expressions or "must include following strings"

Everything else will be empty (not worth the trouble)

For document types I use:
- order confirmations
- Certificates
- Applications
- Demands (taxes, fines, fees, and so on)
- Salary statements
- Credits
- information
- Delivery notices
- Invoices
- Contracts

**Reply by TobenderZephyr** [reply]
Just as a side note:
I make heavy usage of tags. Those are my favorites! I use them for stuff which must be more fine-grained.

If you have had 5 binders (ie. Car, House, Insurance, Work, Finances), I would give each one color.
Car: Dark Green
House: Orange
Insurance: Light Green
Work: Grey
Finances: Light Blue

The same color applies to the tags you would insert into those binders.

Each Family Member gets a tag (White on Black)

A document like payment for rental costs (Electricity) would get those tags:
House (Orange), Electricity (Orange), Todo (Dark Blue) - as long as payment is still due, and "My Name" (White on Black)
